chemprop/v2/cli/predict.py

```diff
@@ -58,7 +59,7 @@ def add_predict_args(parser: ArgumentParser) -> ArgumentParser:
         "--output",
         "--preds-path",
         type=str,
```

Comment:
```suggestion
```

Reply:
why?

Reply:
args are always parsed as strings, so this doesn't do anything